Sources/ConfigManager.swift

```diff
@@ -0,0 +1,46 @@
+/*
+ Copyright 2018 the Material Automation authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+import PerfectLib
+
+struct DefaultConfigParams {
+  static let githubBaseURL = "https://api.github.com"
+  static let projectPath = getProjectDirectory()
+  static let userAgent = "Material Automation"
+  static let helloMessage = "Hello from Material Automation!"
+  static let plistPath = "Config.plist"
```

Comment:
Rad - with this separation it looks like we're getting a step closer to being able to pull the config from the installation repo so that more behavior can be configured by the team that owns the repo.